examples/pytorch/mixhop/main.py

```diff
@@ -207,6 +208,7 @@ def main(args):
                    batchnorm=True)
     
     model = model.to(device)
+    best_model = copy.deepcopy(model)
```

Comment:
`best_model = model` should be fine here

Reply:
I don't think so. We want to save the parameters that perform the best on the validation set.

Reply:
Fine. It's weird to save a model just initialized, to be a candidate of best model.